### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by s-christoff**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+* Add string to `LogType` for more human readable debugging [[GH-442](https://github.com/hashicorp/raft/pull/442)]
+* Extract fuzzy testing into it's own module [[GH-459](https://github.com/hashicorp/raft/pull/459)]
+* Replace `cluster.FailNowf` with `t.Fatalf` [[GH-463](https://github.com/hashicorp/raft/pull/463)]
+
+BUG FIXES
+
+* Increase timeout for `TestRaft_Integ` to make it more reliable [[GH-447](https://github.com/hashicorp/raft/pull/447)]
+* Update LogCache `StoreLogs()` to capture an error that would previously cause a panic [[460](https://github.com/hashicorp/raft/pull/460)]
+* Apply hotfix to `TestRaft_RecoverCluster` to make it more reliable [[GH-461](https://github.com/hashicorp/raft/pull/461)
+* Utilize LeaderCh in `TestRaft_VerifyLeader` to wait for leader response before verifying [[GH-462](https://github.com/hashicorp/raft/pull/462)]
```
Comment (notnoop) [inline CHANGELOG.md:20]: These are internal changes that shouldn't affect the users of the library.  Not sure if it makes sense to include them here.

I don't know if CHANGELOG is meant to capture all changes, or only the changes that are relevant to users.  HashiCorp products only list the user relevant ones in the the changelog, not not plain refactors or test improvements. Though this is a library and maybe has a different standards.

If we choose to keep them, I'd suggest starting INTERNAL section.

Comment (s-christoff) [reply CHANGELOG.md:20]: This makes sense, I'll remove testing updates

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+# 1.3.0 (April 22nd, 2021)
+
 IMPROVEMENTS
 
 * Added metrics for `oldestLogAge` and `lastRestoreDuration` to monitor capacity issues that can cause unrecoverable cluster failure  [[GH-452](https://github.com/hashicorp/raft/pull/452)][[GH-454](https://github.com/hashicorp/raft/pull/454/files)]
 * Made `TrailingLogs`, `SnapshotInterval` and `SnapshotThreshold` reloadable at runtime using a new `ReloadConfig` method. This allows recovery from cases where there are not enough logs retained for followers to catchup after a restart. [[GH-444](https://github.com/hashicorp/raft/pull/444)]
+* Inclusify the repository by switching to main [[GH-446](https://github.com/hashicorp/raft/pull/446)]
+* Add option for a buffered `ApplyCh` if `MaxAppendEntries` is enabled [[GH-445](https://github.com/hashicorp/raft/pull/445)]
+* Add string to `LogType` for more human readable debugging [[GH-442](https://github.com/hashicorp/raft/pull/442)]
+* Extract fuzzy testing into it's own module [[GH-459](https://github.com/hashicorp/raft/pull/459)]
```
Comment (notnoop) [inline CHANGELOG.md:12]:
```suggestion
* Extract fuzzy testing into its own module [[GH-459](https://github.com/hashicorp/raft/pull/459)]
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+
+BUG FIXES
+* Update LogCache `StoreLogs()` to capture an error that would previously cause a panic [[460](https://github.com/hashicorp/raft/pull/460)]
```
Comment (notnoop) [inline CHANGELOG.md:15]:
```suggestion
* Update LogCache `StoreLogs()` to capture an error that would previously cause a panic [[GH-460](https://github.com/hashicorp/raft/pull/460)]
```

**Review by notnoop — APPROVED** [review]
Thanks!

**Pull request merged**